### ISSUE TITLE
IAM user creation for AK/SK user not possible

### ISSUE THREAD

**Issue by Moep90** [opentelekomcloud/terraform-provider-opentelekomcloud#346]
Hi there,

I want to create n users with AK/SK pairs. For example technical users.

I want to create n users with a password. For new colleagues.

In the UI I can choose if a user should be a AK/SK OR a password user.

Unfortunalty I cant choose between these options using terraform.
This might be a limitation within the API since I couldnt find any hint for it in the API-Docs.
So I opened a parallel support request

### Terraform Version
```bash
$ terraform --version
Terraform v0.11.14
```

### Affected Resource(s)
- opentelekomcloud_identity_user_v3

### Terraform Configuration Files
```hcl
resource "opentelekomcloud_identity_user_v3" "user" {
  name               = "${var.username}"
  password           = "${var.password}"
  default_project_id = "${var.default_project_id}"
  domain_id          = "${var.domain_id}"
  enabled            = "${var.enabled}"
}
```

**Comment by niuzhenguo** [comment]
@Moep90 Thanks for raising this,  I also don't find it supported on API side.

**Comment by Moep90** [comment]
I got feedback from the support:
```
[...]
unfortunately we have to inform you that the API doesn't support creating a permanent AK/SK.
For this feature there was already a Feature Request opened. (DM-403)
[...]
If you have any further questions regarding the Feature Request, please do not hesitate to contact us again using the demand number DM-403.
[...]
```

**Comment by Moep90** [comment]
The API seams to be more advanced now:

https://docs.otc.t-systems.com/en-us/api/iam/iam_03_0001.html

**Comment by outcatcher** [comment]
Fixed in #613. Will be included in release v1.20.0

**Comment by outcatcher** [comment]
@Moep90 
Please retest with provider `v1.20.0` ― is that what you expect?
